src/v1/destinations/mp/types.ts

```diff
@@ -0,0 +1,46 @@
+/**
+ * Interface for event object
+ */
+export interface Event {
+  properties?: {
+    $insert_id?: string;
+    [key: string]: any;
+  };
+  [key: string]: any;
+}
+
+/**
+ * Interface for failed record
+ */
+export interface FailedRecord {
+  $insert_id: string;
+  field: string;
+  message: string;
+  [key: string]: any;
+}
+
+/**
```

Comment:
All the below 3 types seems to be generic ones - why they are under mp/types.ts ?